src/SDK/Trace/TracerProvider.php

```diff
@@ -33,7 +33,7 @@ public function __construct(
         }
 
         $spanProcessors = is_array($spanProcessors) ? $spanProcessors : [$spanProcessors];
-        $resource = $resource ?? ResourceInfo::defaultResource();
+        $resource = $resource ?? ResourceInfo::create(new Attributes());
```

Comment:
Why did you change this?

Reply:
So that environment variables are picked up.

Reply:
Hmm, I don't get it, but I'll just believe you :)